lib/std/enums.zig

```diff
@@ -447,6 +453,7 @@ pub fn EnumMap(comptime E: type, comptime V: type) type {
                     result.values[i] = v.*;
                 }
             }
+            return result;
```

Comment:
🤔

Reply:
:thinking: indeed

looks like it was untested and never used!